build/__tools__/make-keys

```diff
@@ -17,23 +17,11 @@
 import argparse
 parser = argparse.ArgumentParser()
 parser.add_argument('--keyfile', help='name of the file to write', default='keyfile')
-parser.add_argument('--format', help='format of the key to generate', choices=['skf', 'pem'])
+parser.add_argument('--format', help='format of the key to generate', choices=['pem'])
 parser.add_argument('--curve', help='ecdsa curve to use for pem keys', default = 'secp256k1', choices=['secp256k1', 'secp384r1'])
 options = parser.parse_args()
```

Comment:
> i suppose ccf assumes secp384?

Correct

> if so, may be we change the default here?

Changing the default implies changing the current behavior here, as well as the crypto lib, updating tests and documentation. Right now this is not critical to support CCF, since we started with the assumption that you must specify the curve in that case.